_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -0,0 +1,5162 @@
+/obj/item/storage/box/coffeepack,
+/obj/item/storage/box/coffeepack,
+/obj/item/storage/box/emptysandbags,
+/obj/item/storage/box/ids,
+/obj/item/storage/box/syndie_kit/chameleon,
+/obj/structure/closet/crate/large{
+	anchored = 1;
+	name = "crew supplies"
+	},
+/obj/item/clothing/head/collectable/xenom,
+/obj/item/xenos_claw,
+/obj/item/toy/toy_xeno,
+/obj/item/clothing/suit/costume/xenos,
+/obj/effect/spawner/random/exotic/antag_gear_weak,
+/obj/effect/spawner/random/exotic/antag_gear_weak,
+/turf/open/floor/plating,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Rw" = (
+/obj/machinery/door/airlock/hatch,
+/obj/structure/cable,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Rx" = (
+/obj/effect/spawner/random/vending/snackvend,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"RG" = (
+/obj/effect/turf_decal/tile/bar{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/bar,
+/obj/item/ammo_casing/spent,
+/obj/item/chair{
+	dir = 1
+	},
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"RK" = (
+/obj/machinery/light/broken/directional/west,
+/obj/machinery/atmospherics/components/unary/vent_pump/on,
+/obj/structure/alien/weeds/node{
+	maximum_growtime = 240000;
+	minimum_growtime = 120000;
+	growtime = 150000
+	},
+/obj/item/ammo_casing/spent,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"RQ" = (
+/obj/structure/fluff/broken_flooring{
+	dir = 4;
+	icon_state = "singular"
+	},
+/turf/open/space/basic,
+/area/template_noop)
+"RS" = (
+/obj/structure/cable,
+/obj/structure/barricade/security,
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/obj/structure/alien/weeds/node{
+	maximum_growtime = 240000;
+	minimum_growtime = 120000;
+	growtime = 150000
+	},
+/obj/item/ammo_casing/spent,
+/obj/item/ammo_casing/spent,
+/obj/item/ammo_casing/spent,
+/obj/item/ammo_casing/spent,
+/obj/structure/door_assembly/door_assembly_hatch{
+	name = "SYN-C Brutus Bridge"
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"RW" = (
+/obj/machinery/door/poddoor/preopen,
+/obj/structure/fans/tiny,
+/obj/effect/mapping_helpers/broken_floor,
+/obj/structure/alien/resin,
+/turf/open/floor/plating,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"RY" = (
+/obj/effect/spawner/random/trash,
+/turf/open/space/basic,
+/area/template_noop)
+"Sr" = (
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 1
+	},
+/turf/closed/wall/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"St" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on{
+	dir = 1
+	},
+/obj/effect/spawner/random/contraband/armory{
```

Comment:
I definitely think you should not be spawning in weapons, remove this please

Reply:
It's inside a sealed and heavily guarded armory on the inside of the heavily guarded ship though!